src/js/control-bar/spacer-controls/insertion-point-spacer.js

```diff
@@ -0,0 +1,23 @@
+import Spacer from './spacer.js';
+
+/**
+ * Spacer specifically meant to be used as an insertion point for new plugins, etc.
+ *
+ * @param {Player|Object} player
+ * @param {Obect=} options
+ */
+class InsertionPointSpacer extends Spacer {
+  buildCSSClass() {
+    return `vjs-insertion-point-spacer-control ${super.buildCSSClass}`;
```

Comment:
We don't actually need `-control` after every class. It started that way because I was expecting every control to inherit from a control class, but that didn't end up happening.